cmd/jwt_test.go

```diff
@@ -35,16 +35,10 @@ func testAuthenticate(authType string, t *testing.T) {
 	}{
 		// Access key too small.
 		{"user", "pass", errInvalidAccessKeyLength},
-		// Access key too long.
-		{"user12345678901234567", "pass", errInvalidAccessKeyLength},
 		// Access key contains unsuppported characters.
 		{"!@#$", "pass", errInvalidAccessKeyLength},
-		// Success when access key contains leading/trailing spaces.
-		{" " + serverCred.AccessKey + " ", serverCred.SecretKey, errInvalidAccessKeyLength},
 		// Secret key too small.
 		{"myuser", "pass", errInvalidSecretKeyLength},
-		// Secret key too long.
-		{"myuser", "pass1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890", errInvalidSecretKeyLength},
```

Comment:
Followings are the reasons for removal of those test cases:
1. 		```
		// Access key too long.
		{"user12345678901234567", "pass", errInvalidAccessKeyLength},
		```
There is no "too long" case after this fix, that is; we don't check max length after removal of max length check. Even if we would like to keep it, test gets `errInvalidAccessKeyID = "The access key ID you provided does not exist in our records"`.
2.		```
 		// Success when access key contains leading/trailing spaces.
		{" " + serverCred.AccessKey + " ", serverCred.SecretKey, errInvalidAccessKeyLength},
		```
This is a left over test after the removal of the cleanup code for leading and trailing spaces in accessKey. (Anyways, this test case was not correct since it was supposed to expect a Pass instead of expecting byte length failure error message.)
3.		```
		// Secret key too long.
		{"myuser", "pass1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890", errInvalidSecretKeyLength},
		```
There is no "too long" case after this fix, removal of max length check.